### PR TITLE
minor fixes on searchElementDialog

### PR DESCRIPTION
**Pull request by AbdelHedhili**
Signed-off-by: Abdelsalem <abdelsalem.hedhili@rte-france.com>

### CODE REVIEW

**Comment by AbdelHedhili** [comment]
Any helpers ? I can't for the life of me align the button on the right so that it's aligned with inputs underneath.

File: src/components/dialogs/input-hooks.js [hunk 1]
```diff
@@ -454,7 +454,7 @@ const DELAY = 1000;
 export const useSearchEquipmentField = ({ handleOpenSearchDialog, label }) => {
```
Comment (sBouzols) [inline src/components/dialogs/input-hooks.js:454]: strange naming. Seems like it's a very generic custom hook. Maybe `useButton` is a better name. Then change `handleOpenSearchDialog` by `handleClick` maybe ?

**Review by sBouzols — COMMENTED** [review]
Code Review ok, but we must change the hook name.
No Test

File: src/components/dialogs/input-hooks.js (hunk 1 above)
Comment (AbdelHedhili) [reply src/components/dialogs/input-hooks.js]: done

**Review by sBouzols — APPROVED** [review]

**Pull request merged**